tests/modules/get_all_professors/app/get_all_professors_usecase.test.ts

```diff
@@ -35,11 +35,11 @@ describe('Assert GetAllProfessorsUsecase is correct at all', () => {
     
     // Validate the suitabilities of the first professor
     const firstProfessorSuitabilities = firstProfessor.suitabilities
-    expect(firstProfessorSuitabilities.length).toEqual(1)
-    firstProfessorSuitabilities.forEach((suitability) => {
-      expect(suitability.codeSubject).toEqual("EFB207")
-      expect(suitability.subjectName).toEqual("Physics I")
-    })
+    expect(firstProfessorSuitabilities.length).toEqual(2)
```

Comment:
I added two new users on mock to for the tests of the get_schedule_by_coordinator, maybe it would be good to merge with dev and see if the tests are going to break because one of them might have suitabilities for these classes